internal/attestation/aws/snp/issuer_test.go

```diff
@@ -40,79 +36,3 @@ func TestGetAttestationKey(t *testing.T) {
 	// if everything worked fine, tpmAk and getAk are the same key
 	assert.Equal(tpmAk, getAk)
 }
-
-func TestGetInstanceInfo(t *testing.T) {
```

Comment:
Should we try to keep testing this function? Happy to help out

Reply:
This functions was removed